src/server/services/afis/afis-facturen.ts

```diff
@@ -260,15 +260,43 @@ function transformFacturen(
 ): AfisFacturenResponse {
   const feedProperties = getFeedEntryProperties(responseData);
   const count = responseData?.feed?.count ?? feedProperties.length;
-  const facturenTransformed = feedProperties.map((invoiceProperties) => {
-    return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
-  });
+  const facturenTransformed = feedProperties
+    .filter((invoiceProperties) => {
+      return shouldFilterOutFactuur(invoiceProperties.PostingDate);
+    })
+    .map((invoiceProperties) => {
+      return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
+    });
   return {
     count,
     facturen: facturenTransformed,
   };
 }
 
+function shouldFilterOutFactuur(postingDate: string): boolean {
+  const postingDateAsDate = new Date(postingDate);
+  const now = new Date();
+
+  if (postingDateAsDate.getTime() > now.getTime()) {
+    captureMessage('postingDate of invoice is in the future', {
+      severity: 'error',
+    });
+    return false;
+  }
+
+  const postedToday =
+    postingDateAsDate.getDate() === now.getDate() &&
+    postingDateAsDate.getMonth() === now.getMonth() &&
+    postingDateAsDate.getFullYear() === now.getFullYear();
+
+  if (!postedToday) {
+    return true;
+  }
```

Comment:
Hier hebben we utility functies voor:

```
if (!isToday(datePosted) && isPast(datePosted)) {
  return true;
}
```

Reply:
heb isToday gebruikt, is dat goed zo? Volgens mij isPast niet helemaal inclusief, m'n test falen als ik !isPast() doe om te checken en ook als ik zo && isPast gebruik.